src/_data/plugins/eleventy-postcss-extension.json

```diff
@@ -0,0 +1,5 @@
+{
+  "npm": "eleventy-postcss-extension",
+  "author": "bart-jaskulski",
```

Comment:
@bart-jaskulski Do you have a Twitter account? I think this is supposed to be a Twitter handle in order get an avatar.

Reply:
@pdehaan Thank you for the insight. Actually, I don't have one. I think we can give it a go without avatar or any link atm :)